src/OrchardCore.Modules/OrchardCore.Users/Workflows/Startup.cs

```diff
@@ -22,6 +22,8 @@ public override void ConfigureServices(IServiceCollection services)
             services.AddActivity<UserLoggedInEvent, UserLoggedInEventDisplayDriver>();
             services.AddScoped<IUserEventHandler, UserEventHandler>();
             services.AddActivity<AssignUserRoleTask, AssignUserRoleTaskDisplayDriver>();
+            services.AddActivity<RemoveUserRoleTask, RemoveUserRoleTaskDisplayDriver>();
```

Comment:
these two should be moved to a new start up class that looks like this instead. this way they are only available when Workflow and roles features are enabled.

```
    [RequireFeatures("OrchardCore.Roles", "OrchardCore.Workflows")]
    public class WorkflowsStartup : StartupBase
    {
        public override void ConfigureServices(IServiceCollection services)
        {
            services.AddActivity<RemoveUserRoleTask, RemoveUserRoleTaskDisplayDriver>();
            services.AddActivity<SelectUsersInRoleTask, SelectUsersInRoleTaskDisplayDriver>();
        }
    }
```